README.md

```diff
@@ -3,7 +3,17 @@
 This repo contains the Docker file used in the [AiiDAlab](https://www.materialscloud.org/aiidalab).
 
 Docker images are available from Dockerhub via `docker pull aiidalab/aiidalab-docker-stack:latest`.
-See [aiidalab/aiidalab-docker-stack](https://hub.docker.com/repository/docker/aiidalab/aiidalab-docker-stack) for a list of available tags.
+
+The available tags and their meaning:
```

Comment:
I suggest to replace line 3 with:
```markdown
This repository contains the Dockerfile for the official AiiDAlab docker images.
```

I suggest to replace lines 5-7 with:
```markdown
Docker images are built and pushed to Docker Hub at https://hub.docker.com/repository/docker/aiidalab/aiidalab-docker-stack with the following tags:
```

Reply:
Just did that. I added the word "automatically" and changed the link. The one you provide requires authentication.